cmd/minishift/cmd/config/view.go

```diff
@@ -47,11 +50,17 @@ var configViewCmd = &cobra.Command{
 	Short: "Display the properties and values of the Minishift configuration file.",
 	Long:  "Display the properties and values of the Minishift configuration file. You can set the output format from one of the available Go templates.",
 	Run: func(cmd *cobra.Command, args []string) {
-		err := configView()
+		cfg, err := ReadConfig()
 		if err != nil {
 			fmt.Fprintln(os.Stderr, err)
 			atexit.Exit(1)
 		}
+		template := determineTemplate(configViewFormat)
+		if err = configView(cfg, template, os.Stdout); err != nil {
+			fmt.Fprintln(os.Stderr, err)
```

Comment:
this will need to be rebased on top of the pull request adding the new -ExitWithMessage_